### PR TITLE
EdkRepo: Add Folder to Folder Cherry Pick

### PR DESCRIPTION
**Pull request by nate-desimone**
Add f2f-cherry-pick command. This command will perform a
cherry-pick while also converting the commit from one
directory heirarchy to another. For example, this command
could be used to cherry-pick commits from KabylakeOpenBoardPkg
to TigerlakeOpenBoardPkg.

Cc: Ashley E Desimone <ashley.e.desimone@intel.com>
Signed-off-by: Nate DeSimone <nathaniel.l.desimone@intel.com>

### CODE REVIEW

File: edkrepo/commands/arguments/f2f_cherry_pick_args.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Copyright (c) 2020 - 2021, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+
+''' Contains the help and description strings for arguments in the
+sync command meta data.
+'''
+
+#Args for the f2f_cherry_pick_command.py
+F2F_CHERRY_PICK_COMMAND_DESCRIPTION = 'Cherry pick changes between different folders within the same branch'
```
Comment (ashedesimone) [inline edkrepo/commands/arguments/f2f_cherry_pick_args.py:15]: Is this accurate? Should it be two folders in the same repository?

File: edkrepo/commands/humble/f2f_cherry_pick_humble.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+#
+## @file
+# f2f_cherry_pick_humble.py
+#
+# Copyright (c) 2020 - 2021, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+
+''' Contains informational and error messages outputted by
+the run_command functions of Edkrepo commands
```
Comment (ashedesimone) [inline edkrepo/commands/humble/f2f_cherry_pick_humble.py:11]: I think this comment should be clarified to say that it is for the EdkRepo f2f command not generic commands

File: edkrepo/commands/arguments/f2f_cherry_pick_args.py (hunk 1 above)
Comment (nate-desimone) [reply edkrepo/commands/arguments/f2f_cherry_pick_args.py]: Please see new patch

File: edkrepo/commands/humble/f2f_cherry_pick_humble.py (hunk 2 above)
Comment (nate-desimone) [reply edkrepo/commands/humble/f2f_cherry_pick_humble.py]: Please see new patch

**Comment by nate-desimone** [comment]
https://github.com/tianocore/edk2-edkrepo/compare/7dc73ac..1f627f9

**Review by ashedesimone — APPROVED** [review]

**Pull request merged**